core/src/trezor/lvglui/scrs/app_passkeys.py

```
@@ -0,0 +1,334 @@
+            self.step3.set_text(f"3. {kwargs['step3']}")
+            self.step3.align_to(self.step2, lv.ALIGN.OUT_BOTTOM_LEFT, 0, 16)
+        if "step4" in kwargs:
+            self.step4 = lv.label(self.content_area)
+            self.step4.add_style(step_text_style, 0)
+            self.step4.set_long_mode(lv.label.LONG.WRAP)
+            self.step4.set_text(f"4. {kwargs['step4']}")
+            self.step4.align_to(self.step3, lv.ALIGN.OUT_BOTTOM_LEFT, 0, 16)
+
+
+class PasskeysRegister(PasskeysGuide):
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)